### PR TITLE
feat: add support case

### PR DESCRIPTION
**Pull request by ralucacril**

### CODE REVIEW

**Review by ufkaya — APPROVED** [review]
LGTM

File: src/services/emailDelivery.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { WithValidationError } from "../types/withValidationError"
+import { ApiCallOptions, handleValidationError, postData } from "../base"
+
+export const supportCases = async ({
+  dealerId,
+  details,
```
Comment (Averethel) [inline src/services/emailDelivery.ts:6]: Is this the body of the message? How about calling it `message`?

File: src/services/emailDelivery.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import { WithValidationError } from "../types/withValidationError"
+import { ApiCallOptions, handleValidationError, postData } from "../base"
+
+export const supportCases = async ({
```
Comment (Averethel) [inline src/services/emailDelivery.ts:4]: Let's go with:

```suggestion
export const sendSupportCase = async ({
```

1. we only send one case, singular reflects that intention
2. let's add `send` prefix to reflect that we're sending some data and be consistent with other naming

**Review by Averethel — APPROVED** [review]
The implementation looks good. I'd just slightly change the naming:

**Pull request merged**